index.js

```diff
@@ -104,6 +105,46 @@ export async function registerSpace (email) {
   console.log(`⁂ space registered to ${email}`)
 }
 
+/**
+ * @param {string} proofPath
+ */
+export async function addSpace (proofPath) {
+  const client = await getClient()
+  const reader = await CarReader.fromIterable(fs.createReadStream(proofPath))
+  const blocks = []
+  for await (const block of reader.blocks()) {
+    blocks.push(block)
+  }
+  // @ts-expect-error
+  const delegation = importDAG(blocks)
```

Comment:
Catch error if DAG is not a delegation?

Reply:
https://github.com/web3-storage/w3cli/issues/15